src/deck.py

```diff
@@ -1,7 +1,27 @@
 from card import Card
+import random
+SUITS = ["C", "D", "H", "S"]
+RANKS = ["2", "3", "4", "5", "6", "7", "8", "9", "10", "J", "Q", "K", "A"]
 
-
+#TODO: implement ranking
 class Deck:
     def __init__(self):
-        print("Deck constructor")
-        self._card = Card()
+        self.cards = []
+        for suit in (SUITS):
+            for rank in (RANKS):
+                card = Card(rank, suit)
+                self.cards.append(card.to_json)
```

Comment:
`card.to_json` needs to be called. This is the reason I was getting blank JSON card objects in my testing.

Reply:
Yep, that's correct. But note the structure of the return as I pointed out above.